core/jobs/src/main/java/com/cognifide/aet/job/common/comparators/jserrors/JsErrorsComparator.java

```diff
@@ -66,19 +66,34 @@ public ComparatorStepResult compare() throws ProcessingException {
         jsErrorLogs = dataFilterJob.modifyData(jsErrorLogs);
       }
       LOGGER.info("Successfully ended data modifications using {}.", comparatorProperties);
-      if (jsErrorLogs.isEmpty()) {
-        result = new ComparatorStepResult(null, ComparatorStepResult.Status.PASSED);
+
+      String artifactId = null;
+      if (!jsErrorLogs.isEmpty()) {
+        artifactId = artifactsDAO.saveArtifactInJsonFormat(comparatorProperties, jsErrorLogs);
+      }
+
+      if (noErrorsOrIgnoredOnly(jsErrorLogs)) {
+        result = new ComparatorStepResult(artifactId, ComparatorStepResult.Status.PASSED);
       } else {
-        final String resultArtifactId = artifactsDAO
-            .saveArtifactInJsonFormat(comparatorProperties, jsErrorLogs);
-        result = new ComparatorStepResult(resultArtifactId, ComparatorStepResult.Status.FAILED);
+        result = new ComparatorStepResult(artifactId, ComparatorStepResult.Status.FAILED);
       }
     } catch (Exception e) {
       throw new ProcessingException("Failed to obtain Js Errors Collection Result!", e);
     }
     return result;
   }
 
+  private boolean noErrorsOrIgnoredOnly(Set<JsErrorLog> jsErrorLogs) {
```

Comment:
It could probably be replaced with:
`return jsErrorLogs.stream().allMatch(JsErrorLog::isIgnored);`